### PR TITLE
doc/tools: refine libnvme.md

### PR DESCRIPTION
**Pull request by zdover23**
This PR refines the language in tools/libnvme.md.

Signed-off-by: Zac Dover <zac.dover@gmail.com>

### CODE REVIEW

File: docs/tools/libnvme.md [hunk 1]
```diff
@@ -6,27 +6,27 @@ sidebar_label: libnvme User Library
...
 
 ## Overview
 
-*libnvme* provides functions for discoverying and managing all nvme devices in
-a linux environment. When the NVMe ZNS specification was ratified, libnvme
-incorporated defintions for all the types and commands that specification
-provides.
+*libnvme* provides functions for discovering and managing all nvme devices in a
+Linux environment. When the NVMe ZNS specification was ratified, libnvme
+incorporated defintions for all the types and commands that the NVMe ZNS
```
Comment (damien-lemoal) [inline docs/tools/libnvme.md:18]: s/defintions/definitions

Comment (zdover23) [reply docs/tools/libnvme.md]: Serves me right for editing while tired.

**Comment by zdover23** [comment]
https://github.com/westerndigitalcorporation/zonedstorage.io/pull/65/files

Back at you, Damien.

You stay up as late as I do.

**Comment by damien-lemoal** [comment]
Applied. Thanks !

**Pull request merged**